docs/architecture/adr-026-rosetta-api.md

```diff
@@ -0,0 +1,136 @@
+# ADR 26: Rosetta API Support
+
+## Changelog
+
+- August 3rd, 2020: Initial Draft.
+
+## Status
+
+- Approved, pending fine tuning the details of implementation.
+
+## Context
+
+We think it'd be greatly valuable to application developers to have the Cosmos SDK
```

Comment:
who is we? ADR's I don't think should specify something like `we`. Just needs reasoning why it will be a good idea

Reply:
Typically, `we` is used in ADRs http://thinkrelevance.com/blog/2011/11/15/documenting-architecture-decisions.

Reply:
ah okay, sorry I found it a bit odd, but ignore my comment